src/lib.rs

```diff
@@ -384,7 +390,9 @@ impl Dependencies {
         let mut flags = BuildFlags::new();
         let mut include_paths = Vec::new();
 
-        for (name, lib) in self.libs.iter() {
+        let mut v = self.libs.iter().collect::<Vec<_>>();
```

Comment:
Can't we use `self.iter()` directly here?

Reply:
Done